internal/upload/koji/koji.go

```diff
@@ -20,8 +23,10 @@ import (
 	"os"
 
 	"github.com/kolo/xmlrpc"
+	logrus "github.com/sirupsen/logrus"
```

Comment:
Just a nitpick, is the `logrus` alias really needed? (maybe it is and I just don't know Go because my IDE is doing this for me :D )